develop/devguide/Automation/UIBlockTypesOverview.md

```diff
@@ -270,6 +270,13 @@ All files uploaded by users will by default be placed in the *C:\\Skyline DataMi
 > [!TIP]
 > See also: [GetUploadedFilePath](xref:Skyline.DataMiner.Automation.UIResults#Skyline_DataMiner_Automation_UIResults_GetUploadedFilePath_System_String_)
 
+> [!NOTE]
+> Contrary to the other UI block types, the FileSelector does not allow to set an initial value. During an interactive automation script session, it is however possible (since RN37620) to keep the files that where already uploaded after the UI was shown.
+>
+> When executing the interactive automation script in a web app, the UI block needs to keep the same row, column and [DestVar](xref:Skyline.DataMiner.Automation.UIBlockDefinition#Skyline_DataMiner_Automation_UIBlockDefinition_DestVar) within the session. If a block of a different type or [DestVar](xref:Skyline.DataMiner.Automation.UIBlockDefinition#Skyline_DataMiner_Automation_UIBlockDefinition_DestVar) is at same location or that location has no blocks when the UI is shown again, the information about the uploaded files is lost.
```

Comment:
"row" and "column" can maybe also be turned into urls that link to the page with more info (like we did for DestVar)

Reply:
They've been added in https://github.com/SkylineCommunications/dataminer-docs/commit/38a9a95e6e7e8f19e2e3b0b22ec45f9d117182bd